string timestamp
float64 map_x
float64 map_y
float64 heading